### DISCUSSION TITLE
Poster

### DISCUSSION THREAD

**Discussion by Pranav400-source** [Tiham2001/tarp#8] [General]
Please suggest some poster templates or how can be make them.

**Comment by Tiham2001** [comment]
We can use templates from https://www.adobe.com/express/create/poster

**Reply by ghost** [reply]
Poster templates to use: [https://moovit.vit.ac.in/mod/assign/view.php?id=52975](https://moovit.vit.ac.in/mod/assign/view.php?id=52975)

**Reply by Pranav400-source** [reply]
Okay sir, thank you.

**Comment by annanya1803** [comment]
We can create posters on Canva and there are predefined templates also to use.

**Reply by Tiham2001** [reply]
Ok
